redhat-appstudio/remote-secret
# Show keys from secret data in the RemoteSecret status

### What does this PR do?
$TITLE

### Screenshot/screencast of this PR

![image](https://github.com/redhat-appstudio/remote-secret/assets/73115616/921b608e-ac83-47c4-be61-2f76672097db)


### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-529

### How to test this PR?

1. Deploy with RS operator: `quay.io/pbaran/rs:SVPI-529` 
2. (checkout this branch) run `./hack/test_remotesecrets.sh`
Before finishing, it will print the RemoteSecret in yaml and it should contain the key `a` and `c`
